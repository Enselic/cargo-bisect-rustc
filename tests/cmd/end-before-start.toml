bin.name = "cargo-bisect-rustc"
args = "--start 2024-03-05 --end 2024-03-01"
status = "failed"
