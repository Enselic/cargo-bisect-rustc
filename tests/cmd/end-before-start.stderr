ERROR: end should be after start, got start: 2024-03-05 and end 2024-03-01
